src/kernels/MIOpenTensorKernels.cl

```diff
@@ -265,83 +206,33 @@ __kernel void OpTensorLeadingOnes(global MIOPEN_TYPE* a,
     global MIOPEN_TYPE* b_off = b + Boffset;
     global MIOPEN_TYPE* c_off = c + Coffset;
 
-#if FIRST_NOT_ONE == 3 // bitmap = 1,1,1,1
-
-    int tid = get_global_id(0);
-    // num_wg: the number of workgroups should be launched
-    // MAX_NUM_WG: the maximum number of workgroups actually launched
-    for(; tid < num_wg; tid += MAX_NUM_WG)
-    {
-
-        MIOPEN_TYPE operand = b_off[tid] * alpha1;
-
-        int o_w = tid % c_w;
-        int o_h = (tid / c_w) % c_h;
-        int o_c = (tid / (c_w * c_h)) % c_c;
-        int o_n = tid / (c_w * c_h * c_c);
-
-        int index    = o_n * c_nstride + o_c * c_cstride + o_h * c_w + o_w;
-        c_off[index] = MIOPEN_TENSOR_OP(a_off[index] * alpha0, operand) + beta * c_off[index];
-    }
-#elif FIRST_NOT_ONE == 2 // bitmap = 1,1,1,0
-    int gid = get_group_id(0);
+    int gid = (bitmap == 15) ? get_global_id(0) : get_group_id(0);
```

Comment:
You could use 0xF - just to hint it is bitmap rather than 15.

Reply:
Will do